docs/getting_started.md

```
@@ -193,3 +191,62 @@ casanovo db-search [PATH_TO_MGF]/sample_preprocessed_spectra.mgf [PATH_TO_FASTA]
 This job should complete in < 1 minute.
 
 Congratulations! Casanovo is installed and running in database searching mode.
+
+### Advanced: Train a new model
+
+Most users of Casanovo will not need to train their own models.
+However, if you have a large collection of annotated spectra and want to try training your own model from scratch, you can run:
+
+```sh
+casanovo train --validation_peak_path validation_spectra.mgf training_spectra.mgf
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)